src/modules/event/fixture/createEvent.ts

```diff
@@ -0,0 +1,24 @@
+import Event, { IEvent } from "../EventModel";
+import { getCounter } from "../../../../test";
+import { DeepPartial } from "../../../../test/deepPartial";
+
+export const createEvent = (args: DeepPartial<IEvent> = {}) => {
+  const i = getCounter("event");
+
+  let { start, end, ...payload } = args;
+
+  if (start === undefined) {
+    start = `2019-01-0${i}T00:00:00.000Z`;
+  }
+
+  if (end === undefined) {
+    end = `2019-01-0${i}T00:00:00.000Z`;
```

Comment:
the end date it will be the same of start if not passing it as params. Do you want this? move at least the end date to the end of the day, so.

`end = `2019-01-0${i}T23:59:59.000Z`;`